pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[project]
+name = "requests-unixsocket"
+version = "0.3.0"
+description = "Use requests to talk HTTP via a UNIX domain socket"
+readme = "README.md"
+requires-python = ">=3.8"
+dependencies = [
+    "requests>=1.1",
+]
+
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm>=8"]
```

Comment:
`setuptools-scm` won't work like this. You have to opt in by adding `[tool.setuptools_scm]` and allowing the version to be dynamic.

Reply:
Yeah I wanted to turn on the `dynamic` stuff too. Thanks for doing it!